aasemble/django/apps/buildsvc/models.py

```diff
@@ -328,6 +328,7 @@ def long_name(self):
     def name(self):
         return self.git_url.split('/')[-1].replace('_', '-')
 
+    @property
     def build(self):
```

Comment:
I *think* just using `.build()` is fine. It's just the `@property` part that is problematic.